content/en/real_user_monitoring/faq/session_replay_service_worker.md

```diff
@@ -17,6 +17,8 @@ If you have blocked third-party cookies in your browser settings, or if your bro
 
 Datadog recommends that you make an exception to your third-party cookie blocking to allow Session Replay's service worker to function properly.
 
+Steps for Chrome are as follows. They are very similar for Firefox and other browsers.
```

Comment:
> and other browsers

Did you check the other browsers? 

Maybe add 

> and other **desktop** browsers

Reply:
Yes. Brave and Edge follow the same pattern as Chrome. Safari doesn't even allow exceptions.